stylesheet.css

```diff
@@ -260,3 +262,41 @@ table tr {
 #tbody tr {
   border: 2px solid black;
 }
+
+@media only screen and (min-width:300px) and (max-width:600px){
+
+  #addButton {
+    left: 4%;
+  }
+  .experience-entry button {
+    left: 178px;
+    top: -289px;
+}
+.navBar ul {
+  padding-left: 0px;
+}
+.navBar ul li {
+  padding: 3px;
+  padding-left: 0px;
+}
+  
+}
+
+@media only screen and (min-width:601px) and (max-width:1024px){
+  #addButton {
+    left: 10%;
+  }
+  
+
+
```

Comment:
remove un-necessary blank spaces from here and check other places too.

Reply:
fixed!